app/services/editor-commands/commands/copy-nodes.ts

```diff
@@ -93,7 +93,7 @@ export class CopyNodesCommand extends Command {
      */
     if (isDualOutputMode && !this.hasNodeMap) {
       // Create all nodes first
-      this.selection.getNodes().forEach(node => {
+      this.selection.getNodes().forEach(async node => {
```

Comment:
What's the purpose of putting the async here?

Reply:
It's to made sure the node and node map entry are completed created so that the dual output source selector icons show correctly